ufm-stream-to-fluentd/README.md

```diff
@@ -51,7 +51,8 @@ Run
 | :---: | :---: |:---: |:---: |
 | --fluentd_host <fluentd_host> | [fluentd-config.host](ufm-stream-to-fluentd.sample.cfg#L19) | True |  Hostname or IP for FluentD endpoint
 | --fluentd_port <fluentd_port> | [fluentd-config.port](ufm-stream-to-fluentd.sample.cfg#L20) | True | Port for FluentD endpoint  
-| --ufm_host <ufm_host> | [ufm-server-config.host](ufm-stream-to-fluentd.sample.cfg#L3) | True | Hostname or IP for The UFM Enterprise
+| --ufm_host <ufm_host> | [ufm-server-config.host](ufm-stream-to-fluentd.sample.cfg#L3) | True | Server name for The UFM Enterprise
```

Comment:
you switched description between ufm_host  and ufm_server_name

Reply:
changed